src/pages/gyeongminLee/Main/Main.js

```diff
@@ -1,15 +1,210 @@
+                />
+                <div id="feedProfileAccount">
+                  {' '}
+                  <span> glorious_min </span>{' '}
+                </div>
+              </div>
+              <div className="feedImage">
+                <img
+                  id="feedImage1"
+                  alt="feed"
+                  src="https://i.pinimg.com/originals/5d/99/89/5d9989c2ddc5aaa8a8627df55c998b62.jpg"
+                />
+              </div>
+
+              <div id="feedFunction">
+                <img
+                  className="feedIcon"
+                  id="likesIcon"
+                  alt="like icon"
+                  src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/heart.png"
+                  alt=""
+                />
+                <img
+                  className="feedIcon"
+                  id="makeCommentIcon"
+                  alt="make comment icon"
+                  src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/comment.png"
+                />
+                <img
+                  className="feedIcon"
+                  id="shareIcon"
+                  alt="share icon"
+                  src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/share.png"
+                />
+              </div>
+
+              <div id="howManyLikes">spanish_ghetto님외 34명이 좋아합니다</div>
+
+              <div id="feedContentsContainer">
+                <span id="feedContentsId">glorious_min</span>
+                <span id="feedContents">사진 찍었다.</span>
+              </div>
+
+              <div className="feedCommentsListContainer">
+                {this.state.list.map((content, id) => {
+                  return <Comment comment={content} key={id} />;
+                })}
```

Comment:
map의 두번째 인자는 index가 들어옵니다 이걸 id라고 이름을 지으면 보는사람이 헷갈릴 수 있겠네요 보통 많이사용하는 index의  축약어인 idx라고 쓰시거나 index라고 전체 다 적어주는게 좋아보입니다!
그리고 key값을 가장 먼저 적어주세요 
```suggestion
                {this.state.list.map((content, idx) => {
                  return <Comment key={idx} comment={content}  />;
                })}
```
그리고 index는 배열의 중간요소가 삭제될 때 변할 수 있기 때문에 유니크하고 변하지 않는 값이여야하는 key값으로는 적절치 않습니다!
다른 값을이용해서 key props부여해주세요